### PR TITLE
Added Tasks for certain build types.

### PR DESCRIPTION
**Pull request by tunjid**
Referencing this [issue](https://github.com/x2on/gradle-hockeyapp-plugin/issues/80)
Tested and works great, currently using it in production with a local jar.

### CODE REVIEW

**Comment by ShreyashPromact** [comment]
can anyone put, what will be the keyword to upload only specific flavor to hockey app?

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Mon Nov 16 17:22:40 CET 2015
+#Wed Mar 30 18:10:21 EDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
```
Comment (x2on) [inline gradle/wrapper/gradle-wrapper.properties]: Please revert this

**Comment by x2on** [comment]
Thanks for your PR.
Please remove all code style changes and then i will merge it.

**Comment by ShreyashPromact** [comment]
Thanks for an update.

**Comment by x2on** [comment]
Any news?

**Comment by x2on** [comment]
So im going to close this because of inactivity.

**Comment by sankartringapps** [comment]
Do we have any plan on getting this feature into the plugin? Am waiting for this for a long time.

**Pull request closed without merging**